src/biokbase/narrative/jobs/job.py

```diff
@@ -396,24 +448,22 @@ def output_state(self, state=None, no_refresh=False) -> dict:
         }
         :rtype: dict
         """
-        if not state:
-            state = self.cached_state() if no_refresh else self.refresh_state()
-        else:
-            self.update_state(state)
-            state = self.cached_state()
-
-        if state is None:
```

Comment:
No way that state can be None with the new logic as `self.cached_state()` fetches `self._acc_state`, which always has something populated.